internal/pkg/dependencies/dependencies.go

```diff
@@ -71,6 +72,7 @@ import (
 // Base contains basic dependencies.
 type Base interface {
 	Envs() env.Provider
+	Tracer() trace.Tracer
```

Comment:
Do `depencies.Base` som pridal `Tracer() trace.Tracer`, z `opentelemetry`.
Pouzivam `opentelemetry` aby sme to mohli vyuzit aj v CLI.
V dalsom PR napojim `opentelemetry` na DD.